src/views/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang = "en">
+
+<head>
+  <!-- Required Meta Tags -->
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- BOOTSTRAP -->
+  <link rel ="stylesheet" type= "text/css" href= "css/main.css">
+
+  
+  <title>Whitney Bowden's Resume Website</title>
+
+</head>
+
+<body>
+    <div class="alert alert-primary" role="alert">
+        This is a primary alert—check it out!
+      </div>
+
+
+    <script src="js/main.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
Cool - you have main.js in here, but we also need to include our vendor scripts. Start with jquery, then include popper, then bootrap, and finally main.js comes last (because the vendor scripts are a dependency). I don't think you need to include the integrity and crossover properties on the script tag.